integration_test/external_tests/commons/test_util.dart

```diff
@@ -62,3 +62,30 @@ Future<void> enterSeeds(AppiumWebDriver driver, String seeds) async {
   var confirmButton = await driver.findElement(confirmButtonLocator);
   await confirmButton.click();
 }
+
```

Comment:
These code is using for static testcase, I think for this PR is not used. please pull code and rebase dev branch